two-photon/rip.py

```diff
@@ -35,10 +35,10 @@ def determine_ripper(data_dir, ripper_dir):
     version = root.attrib['version']
 
     # Prairie View versions are given in the form A.B.C.D.
-    match = re.match(r'^(?P<majmin>\d+\.\d+)\.\d+\.\d+$', version)
+    match = re.match(r'^d+\.\d+\.\d+\.\d+$', version)
     if not match:
         raise RippingError('Could not parse version (expected A.B.C.D): %s' % version)
-    version = match.group('majmin')
+    version = match.group(0)
     ripper = ripper_dir / f'Prairie View {version}' / 'Utilities' / 'Image-Block Ripping Utility.exe'
```

Comment:
The new directories you've added no longer have this pattern -- you've dropped the 'Prairie View ' prefix.

Reply:
You're right! I did that on my local branch but forgot to push it. I'll also be removing the unnecessary files over the next couple days.

Reply:
Okay, I got rid of the extraneous files for the rippers, removed the regex, and corrected the directory used when getting the ripper!